
  0%|                                                                                                                                                                | 0/830 [00:04<?, ?it/s]
Traceback (most recent call last):
  File "src/main.py", line 56, in <module>
    trainer.train(args.epochs)
  File "/home/gabriel/Documents/[001]Projects/CGIAR-UPCH/repos/cgiar-malaria/src/trainer.py", line 59, in train
    train_loss = self.train_epoch(self.loaders['train'])
  File "/home/gabriel/Documents/[001]Projects/CGIAR-UPCH/repos/cgiar-malaria/src/trainer.py", line 37, in train_epoch
    pred = self.model(img)
  File "/home/gabriel/.local/share/virtualenvs/cgiar-malaria-KE3vYxy0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/gabriel/Documents/[001]Projects/CGIAR-UPCH/repos/cgiar-malaria/src/models/unet.py", line 64, in forward
    decoder_outputs.append(op(x))
  File "/home/gabriel/.local/share/virtualenvs/cgiar-malaria-KE3vYxy0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/gabriel/Documents/[001]Projects/CGIAR-UPCH/repos/cgiar-malaria/src/models/unet.py", line 16, in forward
    x = F.relu(self.conv1(x))
  File "/home/gabriel/.local/share/virtualenvs/cgiar-malaria-KE3vYxy0/lib/python3.6/site-packages/torch/nn/functional.py", line 1299, in relu
    result = torch.relu(input)
RuntimeError: CUDA out of memory. Tried to allocate 128.00 MiB (GPU 0; 3.94 GiB total capacity; 1.30 GiB already allocated; 95.19 MiB free; 1.38 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF